src/compas_fab/robots/robot.py

```diff
@@ -871,48 +1037,61 @@ def position_constraint_from_frame(self, frame_WCF, tolerance_position, group=No
         frame_WCF : :class:`compas.geometry.Frame`
             The frame from which we create position and orientation constraints.
         tolerance_position : float
-            The allowed tolerance to the frame's position. (Defined in the
-            robot's units)
-        group: str
+            The allowed tolerance to the frame's position (defined in the
+            robot's units).
+        group: str, optional
             The planning group for which we specify the constraint. Defaults to
             the robot's main planning group.
 
+        Returns
+        -------
+        :class:`PositionConstraint`
+
+        See Also
+        --------
+        * :meth:`PositionConstraint.from_box`
+        * :meth:`PositionConstraint.from_mesh`
+        * :meth:`PositionConstraint.from_sphere`
+
         Examples
         --------
         >>> frame = Frame([0.4, 0.3, 0.4], [0, 1, 0], [0, 0, 1])
         >>> tolerance_position = 0.001
         >>> robot.position_constraint_from_frame(frame, tolerance_position)
         PositionConstraint('ee_link', BoundingVolume(2, Sphere(Point(0.400, 0.300, 0.400), 0.001)), 1.0)
-
-        Notes
-        -----
-        There are many other possibilities of how to create a position and
-        orientation constraints. Checkout :class:`compas_fab.robots.PositionConstraint`
-        and :class:`compas_fab.robots.OrientationConstraint`.
-
         """
-
         ee_link = self.get_end_effector_link_name(group)
         sphere = Sphere(frame_WCF.point, tolerance_position)
         return PositionConstraint.from_sphere(ee_link, sphere)
 
     def constraints_from_frame(self, frame_WCF, tolerance_position, tolerances_axes, group=None):
-        """Returns a position and orientation constraint on the group's end-effector link.
+        """Returns a position and an orientation constraint on the group's end-effector link.
 
         Parameters
         ----------
         frame_WCF: :class:`compas.geometry.Frame`
             The frame from which we create position and orientation constraints.
         tolerance_position: float
-            The allowed tolerance to the frame's position. (Defined in the
-            robot's units)
+            The allowed tolerance to the frame's position (defined in the
+            robot's units).
         tolerances_axes: list of float
-            Error tolerances ti for each of the frame's axes in radians. If only
+            Error tolerances *ti* for each of the frame's axes in radians. If only
```

Comment:
What means `ti`?

Reply:
No idea, I've tried googling it but haven't figured it out. Thought it was an engineering thing ;).

It's present in three lines on master:
```
$ rg " ti "
src/compas_fab/robots/constraints.py
199:        Error tolerances ti for each of the frame's axes. The respective

src/compas_fab/robots/robot.py
835:            Error tolerances ti for each of the frame's axes in radians. If only
910:            Error tolerances ti for each of the frame's axes in radians. If only
```

Reply:
@romanarust : do you remember if this `ti` thing means anything here or if it's a typo?

Reply:
no typo (tolerance_index), but maybe it should be named t_i (i subscript), i in range of [0,2] for each of the frame's axes (t_x, t_y, t_z)

Reply:
Ok! I can write inline LaTeX to do the subscript and add the your explanation then :). Maybe a footnote might be good, do you have any recommendations for where to find more information about this?

Reply:
No, unfortunately badly documented:
see `optional axis-angle error tolerances specified`
http://docs.ros.org/melodic/api/moveit_msgs/html/msg/OrientationConstraint.html

Reply:
d421722